public/styles/components/campaign-info/analytics/_analytics.scss

```diff
@@ -0,0 +1,72 @@
+.campaign-info {
+  width: 100%;
+
+}
+
+.analytics-chart__full-width {
+
```

Comment:
This is exactly the kind of help I need with the scss, I know what I'm doing is probably not idiomatic (even though it appears to work).

Also naming classes, never really got my head round all that __ stuff.